package.json

```diff
@@ -40,6 +40,7 @@
 		"json"
 	],
 	"devDependencies": {
+		"@types/node": "^17.0.41",
```

Comment:
Re: this change, it looks like there was a breaking change to this package - see [this stackoverflow post](https://stackoverflow.com/questions/75386478/error-in-node-modules-types-node-ts4-8-util-d-ts148542-error-ts1005-ex). Happy to adopt another approach if you want.

Also - happy to uprev the min version to node 8 if you're so inclined for deprecation/unit test reasons, but didn't want to do it unilaterally.